src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -3290,3 +3290,24 @@ export function getLimitValues(data: any[]): string[] {
         .uniq()
         .value();
 }
+
+export function stringsToOptions(vals: string[]) {
```

Comment:
seems like it's only used here so maybe don't export?  kind of a niche operation.